graphql/schema.graphql

```diff
@@ -331,6 +331,12 @@ type SimpleRole {
   slug: String
 }
 
+type B2BSettings {
+  autoApprove: Boolean
+  defaultPaymentTerms: PaymentTerm[]
```

Comment:
Shouldn't this be `[PaymentTerm]`?

Reply:
i'll change it back, it was originally

Reply:
Did I tell you to do it the other way before? If so my bad, I was confused between Typescript and GraphQL. In TS the brackets need to go at the end, with GraphQL they wrap the type.